src/DynamoCoreWpf/Controls/InfiniteGridView.xaml.cs

```diff
@@ -44,8 +44,7 @@ public GridVisualHost()
             minorGridPen = new Pen(minorBrush, 1.0);
 
             AddVisualChild(drawingVisual);
-            this.SizeChanged += (s, e) => UpdateDrawingVisual();
-            this.Loaded += (s, e) => InitializeOnce();
+            SizeChanged += (s, e) => UpdateDrawingVisual();
```

Comment:
`Loaded` event occurs after the first time of `SizeChanged`. That is why initial drawing doesn't take place and on Dynamo start blank background is shown